docker-compose.preview.yml

```diff
@@ -0,0 +1,44 @@
+services:
+    platform:
+        image: PLATFORM_IMAGE
+        environment:
+            POSTGRES_USER: preview
+            POSTGRES_PASSWORD: preview
+            POSTGRES_DB: preview
+            MINIO_ROOT_USER: preview
+            MINIO_ROOT_PASSWORD: preview
+            ASSETS_UPLOAD_KEY: preview
+            ASSETS_UPLOAD_SECRET_KEY: preview
+    platform-jobs:
+        image: JOBS_IMAGE
+    platform-migrations:
+        image: MIGRATIONS_IMAGE
+    caddy:
+        image: caddy:latest
+        restart: unless-stopped
+        command: "caddy reverse-proxy --from '${PULLPREVIEW_PUBLIC_DNS}' --to platform:3000"
+        depends_on:
+            - platform
+            - platform-jobs
+            - minio
+        ports:
+            - "80:80"
+            - "443:443"
+        volumes:
+            - ./caddy:/etc/caddy
+            - caddy-data:/data
+            - caddy-config:/config
+        networks:
+            - app-network
+    seed:
+        platform: linux/amd64
+        depends_on:
+            platform:
+                condition: service_started
+        image: ghcr.io/pubpub/platform-migrations:latest
+        env_file: .env
+        environment:
+            DATABASE_URL: postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
+        command: ["pnpm", "--filter", "core", "reset"]
+        networks:
+            - app-network
```

Comment:
maybe an even simpler setup would be to just have the migrations container also run the reset, i don't think there's any real problem with that.

so you'd remove this, and to `platform-migrations` add (untested)

```yaml
command: ["pnpm", "--filter", "core", "migrate-docker", "&&", "pnpm", "--filter", "core", "reset"]
```